packages/cyberstorm/src/components/Dialog/Dialog.tsx

```diff
@@ -108,28 +78,12 @@ export function Dialog(props: DialogProps) {
               <div
                 className={classnames(
                   styles.body,
-                  noPadding ? null : styles.bodyPadding
+                  noPadding ? null : styles.bodyPadding,
+                  showFooterBorder ? null : styles.footerBorder
```

Comment:
TBH what you're trying to do with this prop seems really hacky. Can you come up with alternative ways to achieve what you're aiming for?

Reply:
Now there's some `preStyledFooter` prop that isn't used?